slack-lint-checks/src/main/java/slack/lint/denylistedapis/DenyListedEntryLoader.kt

```diff
@@ -0,0 +1,5 @@
+package slack.lint.denylistedapis
+
+interface DenyListedEntryLoader {
```

Comment:
Would you mind adding an experimental annotation for this? None of this is a stable API anyway, but want to make it doubly-clear here :). Can call it `ExperimentalSlackLintApi`

Reply:
Let's also name this `DenyListedEntryProvider` and make it have a function instead (i.e. `getEntries()`) that takes a `JavaContext`. Then let's lazily load these in the element handler and provide the available context to it, so we can minimize loading (especially if there are entries provided that don't apply in certain contexts).

Reply:
oh yeah sure, good call.